packages/api/src/graphql/image/image.queries.ts

```diff
@@ -0,0 +1,105 @@
+import {Image, Prisma, PrismaClient} from '@prisma/client'
+import {ConnectionResult, MaxResultsPerPage} from '../../db/common'
+import {ImageFilter, ImageSort} from '../../db/image'
+import {getSortOrder, SortOrder} from '../queries/sort'
+
+export const createImageOrder = (
+  field: ImageSort,
+  sortOrder: SortOrder
+): Prisma.ImageFindManyArgs['orderBy'] => {
+  switch (field) {
```

Comment:
Couldn't it be much more easier like this:
`return {field: sortOrder}`

We wouldn't even use the switch statement therefor

Reply:
This is to keep them consistent as some orders return objects like 

```
{
    published: {
      updatedAt: sortOrder
    }
  }
```